src/Illuminate/Database/Query/Grammars/MySqlGrammar.php

```diff
@@ -524,4 +524,19 @@ protected function wrapJsonBooleanSelector($value)
 
         return 'json_extract('.$field.$path.')';
     }
+
+    /**
+     * Apply custom ordering to a query based on a priority array.
+     *
+     * @param  $column
+     * @param  array  $priority
+     * @param  $direction
+     * @return string
+     */
+    public function orderByPriority($column, array $priority, $direction = 'asc')
+    {
+        $placeholders = implode(',', array_fill(0, count($priority), '?'));
+
+        return "FIELD($column, $placeholders) $direction";
```

Comment:
The column name needs to be wrapped with `$this->wrap($column)`. This also affects the other grammars.

Reply:
I have applied the wrap function on all grammars and updated the tests accordingly.